QgisModelBaker/libqgsprojectgen/dbconnector/mssql_connector.py

```diff
@@ -544,3 +548,15 @@ def ili_version(self):
             return 3
         else:
             return 4
+
+    def get_basket_handling(self):
+        if self.schema and self._table_exists(SETTINGS_TABLE):
+                cur = self.conn.cursor()
+                cur.execute("""SELECT setting
+                            FROM {schema}.{settings_table}
+                            WHERE tag = 'ch.ehi.ili2db.BasketHandling'
+                            """.format(schema=self.schema, settings_table=SETTINGS_TABLE))
+                content = cur.fetchone()
+                if content:
+                    return content[0] == 'readWrite'
```

Comment:
Where does `readWrite` come from?

Reply:
When `--createBasketCol` is set, this value is `readWrite`
I don't know if it ever diverts from that (besides of this column is not existing when `--createBasketCol` is not set).
Anyway I made a search in the ili2db code and I only find tests considering the `readWrite` value and no other.

So I think it's most stable to consider this value to get the information, if there is a basket handling or not...